worlds/hylics2/Rules.py

```diff
@@ -129,6 +129,12 @@ def set_rules(hylics2world):
     world = hylics2world.multiworld
```

Comment:
Probably for a different PR, you should probably change this to `multiworld = hylics2world.multiworld`. It's being progressively pushed to stop doing `world = multiworld` since it can get confusing sometimes, especially for new devs looking at other world implementations.